server/Makefile

```diff
@@ -41,7 +44,7 @@ install-python3-setup: install-bin install-lib
 	mkdir -p ${DESTDIR}/python3
 	${COPY} requirements.txt ${DESTDIR}
 	(cd ..; SKIP_GENERATE_AUTHORS=1 SKIP_WRITE_GIT_CHANGELOG=1 python3 setup.py install --prefix=${DESTDIR}/python3)
-	${COPY} ${DESTDIR}/python3/bin/pbench-config ${DESTDIR}/python3/bin/pbench-server ${BINDIR}/
+	${COPY} ${DESTDIR}/python3/bin/pbench-config ${DESTDIR}/python3/bin/pbench-server ${click-scripts} ${BINDIR}/
```

Comment:
I don't think that this will actually work:  I don't think `pbench-user-create` is in the current working directory when this copy command is executed.

In the `agent/Makefile`, we prefix the contents of the list with the appropriate directory path:
```
$(addprefix ${DESTDIR}/python3/bin/, ${click-scripts})
```
I expect that you'll need to do that here.